src/application.py

```diff
@@ -48,6 +48,7 @@ def _setup_accelerators(self):
         self.set_accels_for_action("win.pencil-toggled", ["p"])
         self.set_accels_for_action("win.back-to-menu", ["<Ctrl>m"])
         self.set_accels_for_action("win.show-primary-menu", ["F10"])
+        self.set_accels_for_action("win.show-help-overlay", ["<Ctrl>question", "<Ctrl>slash"])
```

Comment:
I understand what you're trying to accomplish here, but all GNOME apps that I've tested only use `"<Ctrl>question"`, not `"<Ctrl>slash"`, and if we do otherwise, it might get flagged by the reviewers at GNOME Circle.
```suggestion
        self.set_accels_for_action("win.show-help-overlay"])
```